maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -847,12 +847,31 @@ private DefaultScanResult scanDependencies()
         {
             try
             {
+                DefaultScanResult scanResult = new DefaultScanResult( Collections.EMPTY_LIST );
```

Comment:
Do you use IntelliJ IDEA? It highlights the argument in yellow.
Use this:
`DefaultScanResult scanResult = new DefaultScanResult( Collections.<String>emptyList() );`